client_interface.py

```diff
@@ -51,12 +53,17 @@ def send_to_server(state):
     # create Session
     async with aiohttp.ClientSession() as session:
         # create Websocket
-        async with session.ws_connect('http://localhost:8080/ws/') as ws:
+        async with session.ws_connect('http://localhost:8080/interface/') as ws:
             async for msg in ws:
                 # Cycle "for" is finished when client disconnect from server
                 if msg.type == aiohttp.WSMsgType.TEXT:
-                    message = msg.data
-                    print(message)
+                    if msg.data.startswith("robot"):
```

Comment:
Proč je to tu rozděleno?

Reply:
Protoze chceme mit za tim nejakou logiku: obdrzena informace o robotovi ma zpusobit vykresleni obrazku robota a nacteni jeho parametru (asi tim vytvoreni instance objektu Robot), zatimco karty se vykresli jinde a budeme s nimi provadet jine cinnosti.

Reply:
Vubec mi nedoslo ze se to bude načítat znovu celé i ten robot  😂 porad ze jen ty karty.